operate/schema/src/main/java/io/camunda/operate/schema/elasticsearch/ElasticsearchSchemaManager.java

```diff
@@ -92,6 +92,23 @@ public void createSchema() {
     createIndices();
   }
 
+  @Override
+  public void checkAndUpdateIndices() {
+    LOGGER.info("Updating Indices with currently-configured number of replicas...");
+    final String currentConfigNumberOfReplicas =
+        String.valueOf(operateProperties.getElasticsearch().getNumberOfReplicas());
```

Comment:
We also have another way to configure number of replicas: [map of values per index name](https://github.com/camunda/zeebe/blob/64b371c8ae08023f384726673b7d645795f2efa5/operate/common/src/main/java/io/camunda/operate/property/OperateElasticsearchProperties.java#L32C32-L32C59). Do we also want to take this into account? You could check how two properties are used when creating the schema, which has precedence etc.

Reply:
The getter of the map is used in the [schema manager](https://github.com/camunda/zeebe/blob/64b371c8ae08023f384726673b7d645795f2efa5/operate/schema/src/main/java/io/camunda/operate/schema/opensearch/OpensearchSchemaManager.java#L347), but it is never being set anywhere. I'll set it up to try to use that value in case it does get used.

Reply:
It can be when configuring the application. See examples here: https://github.com/camunda/operate/issues/5920
I will write separate comment, as I see even bigger issue here.